swift/Cartfile

```diff
@@ -1,2 +1,2 @@
 github "MessageKit/MessageKit" ~> 3.0
-github "zeroc-ice/ice" ~> 3.7
+github "zeroc-ice/ice" "3.7"
```

Comment:
what's the difference between `~> 3.7` and `"3.7"`

Reply:
yes see https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#version-requirement